resubmit.py

```diff
@@ -251,9 +209,9 @@ def extendWorkflow(workflow, user, group, verbose=False, events=None, firstlumi=
     firstlumi = int(firstlumi)
 
     # Get info about the workflow to be cloned
-    helper = reqMgrClient.retrieveSchema(workflow)
-    schema = modifySchema(helper, workflow, user, group, None, events, firstlumi, None)
-    schema['OriginalRequestName'] = workflow
+    cache = reqMgrClient.getWorkflowInfo(url, workflow)
```

Comment:
@amaltaro I think we have been back and forth a couple of times on using the Info instead of the cache because of missing items in the "info". Can you recall adding everything that had been missing ?
@prozober if I recall correctly, some of the missing parameters were silently not making things fail, but the clone would not have the expected parameters, and use some default values instead.